bin/cqlsh.py

```diff
@@ -2107,16 +2117,47 @@ def should_use_color():
     return True
 
 
+def is_file_secure(filename):
+    if is_win:
+        # We simply cannot tell whether the file is seucre on Windows,
```

Comment:
That's a bummer, but could we at least notify the user that we can't determine the safety? I just think about the `os.chmod` call in Python and how it silently fails on Windows

Reply:
I think that notification will persistently hunt all Windows CQLSH users and can be a source of annoyance for many. I would rather try to think about the bright side, Windows is rarely used in multi-user environments, and file ownership and permission usually aren't a concern on such system. In case a Windows machine is shared between multiple users, it's extremely likely all users (and future users) would have physical access to the machine. File ownership and permission are much less of a concern if the physical security of the machine simply cannot be guaranteed.